docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -15,6 +15,7 @@ Using the command line Roslyn can be developed using the following pattern:
     - Ensure C#, VB, MSBuild and Visual Studio Extensibility are included in the selected work loads
     - Ensure Visual Studio is on Version "15.3" or greater
 1. [.NET Core SDK 2.0](https://www.microsoft.com/net/download/core)
+1. If you're on Windows 7, you need to update to at least PowerShell 3.0. You can find the upgraded links at [Installing Windows PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-windows-powershell) under "upgrading existing Windows PowerShell".
```

Comment:
The format on this list is product name then justification. Should match that here

Reply:
Actually i am not even sure i want to say this much. The reality is that we do not officially support Windows 7 as a dev platform for this repo. We take low impact bug fixes but do 0 work to ensure it keeps working. This statement makes it appear that we support it officially.

Reply:
@jaredpar We may not be officially supporting it, but not saying this means are then paying a support cost with this question every month or so. So ironically the less we say creates more support cost for Windows 7!

Reply:
Seeing this line would have saved me a couple of hours of time this afternoon... any reason this hasn't been merged in yet?